subiquity/common/types.py

```diff
@@ -268,6 +268,7 @@ class Partition:
     boot: Optional[bool] = None
     os: Optional[OsProber] = None
     offset: Optional[int] = None
+    estimated_min_size: Optional[int] = -1
```

Comment:
If I understand correctly, this should never be None but I might have overlooked something

Reply:
The Partition object is a bit different as it has two major use cases
* Originating from Subiquity, in which case we're going to give all the fields reasonable values.
* Originating from a client, which is only going to fill out a few fields.

While we could ask clients to send a slightly different object with just the required fields, doing it this way allows the client to get a Partition from a previous call, modify it, then send it back.
Also, the serializer objects if there are extra fields.

Reply:
Thanks for the details! I understand the use-cases better.
Would None be a valid value when the Partition objects originates from Subiquity? or when it originates from the client?
I was expecting the value -1 to be set by the deserializer if `estimated_min_size` is absent from the request, is that incorrect?